js/core/issues-notes.js

```diff
@@ -67,20 +72,47 @@ define(
                                 }
                             }
                             $tit.find("span").text(text);
-                            report.title = $inno.attr("title");
+                            report.title = $inno.attr("title") || (ghIssue && ghIssue.title) || null;
                             if (report.title) {
                                 $tit.append(doc.createTextNode(": " + report.title));
                                 $inno.removeAttr("title");
                             }
                             $div.append($tit);
                             $inno.replaceWith($div);
-                            $div.append($inno.removeClass(report.type).removeAttr('data-number'));
+                            var body = $inno.removeClass(report.type).removeAttr('data-number');
+                            if (ghIssue && !body.text().trim()) {
+                                body = marked(ghIssue.body, {
+                                    gfm: true,
+                                    pedantic: false,
```

Comment:
Dude, I actively practice cargo-culting. They're supposed to do wonderful things. In practice, I'm not so sure.